ios/FluentUI.Demo/FluentUI.Demo/Demos/DividerDemoController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class DividerDemoController: DemoController {
```

Comment:
I think a SwiftUI demo controller could be a good showcase of the control directly in SwiftUI.